app/models/status.rb

```diff
@@ -47,6 +47,7 @@ class Status < ApplicationRecord
   attr_accessor :override_timestamps
 
   update_index('statuses', :proper)
+  update_index('publicStatuses', :proper)
```

Comment:
Does Chewy really use camel-case for constant resolution? It feels like snake_case would be more natural. Does it maybe do both?

Reply:
Good call.